### PR TITLE
Translate the news "Ruby 2.2.2 Released" (zh_cn)

### PR DESCRIPTION
**Pull request by happybai**
Related #1074

### CODE REVIEW

File: zh_cn/news/_posts/2015-04-13-ruby-2-2-2-released.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+author: "nagachika"
+translator: "happybai"
+date: 2015-04-13 12:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布发布 Ruby 2.2.2。
+这是 Ruby 2.2 的一个补丁版本
+
+这次发布包含了修复 OpenSSL 扩展的主机名验证漏洞
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-04-13-ruby-2-2-2-released.md]: missing a chinese period: `。`.

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi

Please review this pull request, thanks!

**Comment by hlcfan** [comment]
Looks good!

File: zh_cn/news/_posts/2015-04-13-ruby-2-2-2-released.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+---
+
+我们很高兴地宣布发布 Ruby 2.2.2。
+这是 Ruby 2.2 的一个补丁版本。
+
+这次发布包含了修复 OpenSSL 扩展的主机名验证漏洞。
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
+
+以及大量的 Bug 修正。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-04-13-ruby-2-2-2-released.md]: "some" does not mean "大量的", IMO.

File: zh_cn/news/_posts/2015-04-13-ruby-2-2-2-released.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+详见 [ChangeLog](http://svn.ruby-lang.org/repos/ruby/tags/v2_2_2/ChangeLog)
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-04-13-ruby-2-2-2-released.md]: Missing period.

**Comment by happybai** [comment]
Updated, please review.

**Comment by JuanitoFatas** [comment]
Thanks! @happybai I have fixed some minor things and merged manually as 39cf79c.

**Pull request closed without merging**